### ISSUE TITLE
kill Italic in examples

### ISSUE THREAD

**Issue by VladimirAlexiev** [validatingrdf/validatingrdf.github.io#5]
I think the italic in examples (eg http://book.validatingrdf.com/bookHtml010.html#ch040%3AShExSimpleExample) makes them harder to read. Please use upright font, the cyan background is more than enough to show that it's an example

**Comment by labra** [comment]
Indeed, the italics is not on purpose. I need to dig a little bit in Hevea to know why it added the italics to the example environment
